src/components/BottomBarTabView.vue

```diff
@@ -234,7 +235,7 @@ export default {
     }
   }
 }
-@media only screen and (max-width: 878px) {
+@media only screen and (max-width: 600px) {
```

Comment:
I'm seeing 976 and 440 breakpoints elsewhere. Is having a third one necessary?

Reply:
At the 600px breakpoint: Adjusting for smaller mobile size, see more tab width. 

I updated the PR description up top to include the reasoning for additional breakpoints if you want me to expand more on them :)